src/task-completion/index.tsx

```diff
@@ -0,0 +1,279 @@
+				);
+				const loadingOptions =
+					! hasFinishedResolution( 'getOption', [
+						SHOWN_FOR_ACTIONS_OPTION_NAME,
+					] ) ||
+					! hasFinishedResolution( 'getOption', [
+						ADMIN_INSTALL_TIMESTAMP_OPTION_NAME,
+					] );
+				return {
+					storeAgeInWeeks: getStoreAgeInWeeks(
+						adminInstallTimestamp
+					),
+					cesShownForActions: cesActions,
+					canShowCustomerEffortScore:
+						! loadingOptions &&
+						allowTracking &&
+						! ( cesActions || [] ).includes( 'store_setup' ),
+					loading: loadingOptions,
+				};
+			}
+			return {};
+		} );
+
+	useEffect( () => {
+		if ( hasSubmittedScore ) {
+			setTimeout( () => {
+				setHideCustomerEffortScore( true );
+			}, 1200 );
+		}
+	}, [ hasSubmittedScore ] );
+
+	const submitScore = ( {
+		firstScore,
+		secondScore,
+		comments,
+	}: {
+		firstScore: number;
+		secondScore?: number;
+		comments?: string;
+	} ) => {
+		recordEvent( 'ces_feedback', {
+			action: CUSTOMER_EFFORT_SCORE_ACTION,
+			score: firstScore,
+			score_second_question: secondScore ?? null,
+			score_combined: firstScore + ( secondScore ?? 0 ),
+			comments: comments || '',
+			store_age: storeAgeInWeeks,
+		} );
+		updateOptions( {
+			[ SHOWN_FOR_ACTIONS_OPTION_NAME ]: [
+				CUSTOMER_EFFORT_SCORE_ACTION,
+				...( cesShownForActions || [] ),
+			],
+		} );
+		setHasSubmittedScore( true );
+	};
+
+	const recordScore = ( recordedScore: number ) => {
+		if ( recordedScore > 2 ) {
+			setScore( recordedScore );
+			submitScore( { firstScore: recordedScore } );
+		} else {
+			setScore( recordedScore );
+			setShowCesModal( true );
+			recordEvent( 'ces_view', {
+				action: CUSTOMER_EFFORT_SCORE_ACTION,
+				store_age: storeAgeInWeeks,
+			} );
+		}
+	};
+
+	const recordModalScore = (
+		firstScore: number,
+		secondScore: number,
+		comments: string
+	) => {
+		setShowCesModal( false );
+		submitScore( { firstScore, secondScore, comments } );
+	};
+
+	return (
+		<>
+			<div
+				className={ classnames(
+					'woocommerce-task-dashboard__container two-column-experiment'
+				) }
+			>
+				<Card
+					size="large"
+					className="woocommerce-task-card woocommerce-homescreen-card completed"
+				>
+					<CardHeader size="medium">
+						<div className="wooocommerce-task-card__header">
+							<HeaderImage
+								className="wooocommerce-task-card__finished-header-image"
+								alt="Completed"
+							/>
+
+							<Text size="title" as="h2" lineHeight={ 1.4 }>
+								{ __(
+									'Woohoo! Your trial store has been set up!',
+									'wc-calypso-bridge'
+								) }
+							</Text>
+							<Text
+								variant="subtitle.small"
+								as="p"
+								size="13"
+								lineHeight="16px"
+								className="wooocommerce-task-card__header-subtitle"
+							>
+								{ __(
+									"Congratulations! Take a moment to celebrate. And once you're ready to launch your store, all you have to do is sign up for a plan. This will also unlock the two upcoming tasks in the task list.",
```

Comment:
```suggestion
									"Congratulations! Take a moment to celebrate. And once you're ready to launch your store, all you have to do is upgrade to a paid plan. This will also unlock the two upcoming tasks in the task list.",
```